daemon/gluster-blockd.c

```diff
@@ -61,7 +61,7 @@ glusterBlockCliThreadProc (void *vargp)
 {
   register SVCXPRT *transp = NULL;
   struct sockaddr_un saun = {0, };
-  int sockfd = -1;
+  int sockfd = RPC_ANYSOCK;
```

Comment:
Should we also change the code like:

   if (sockfd != -1) {  ## -1 --> RPC_ANYSOCK
     close(sockfd);	   
   }
??

Reply:
Done